### PR TITLE
remove unnecessary slash from regex

### PR DESCRIPTION
**Pull request by nickpell**
Gets rid of warnings in django 1.9:

```
WARNINGS:
?: (urls.W002) Your URL pattern '^/(?P<app_id>[\w-]+)/$' [name='receiver_post_with_app_id'] has a regex beginning with a '/'. Remove this slash as it is unnecessary.
?: (urls.W002) Your URL pattern '^/?$' [name='receiver_post'] has a regex beginning with a '/'. Remove this slash as it is unnecessary.
?: (urls.W002) Your URL pattern '^/secure/$' [name='receiver_secure_post'] has a regex beginning with a '/'. Remove this slash as it is unnecessary.
?: (urls.W002) Your URL pattern '^/secure/(?P<app_id>[\w-]+)/$' [name='receiver_secure_post_with_app_id'] has a regex beginning with a '/'. Remove this slash as it is unnecessary.
?: (urls.W002) Your URL pattern '^/submission/?$' [name='receiver_odk_post'] has a regex beginning with a '/'. Remove this slash as it is unnecessary.
```

@calellowitz

### CODE REVIEW

**Comment by calellowitz** [comment]
👍 on ✅

**Comment by czue** [comment]
build fail looks related
```
FAIL: testExportFilter (corehq.apps.reports.tests.test_export_api.ExportTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/reports/tests/test_export_api.py", line 176, in testExportFilter
    self.assertEqual(200, resp.status_code)
AssertionError: 200 != 404
```

**Comment by czue** [comment]
(there are others as well)

File: corehq/apps/receiverwrapper/urls.py [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
     url(r'^/?$', 'post', name='receiver_post'),
-    url(r'^/secure/(?P<app_id>[\w-]+)/$', 'secure_post', name='receiver_secure_post_with_app_id'),
-    url(r'^/secure/$', 'secure_post', name='receiver_secure_post'),
+    url(r'^secure/(?P<app_id>[\w-]+)/$', 'secure_post', name='receiver_secure_post_with_app_id'),
+    url(r'^secure/$', 'secure_post', name='receiver_secure_post'),
 
     # odk urls
-    url(r'^/submission/?$',  'post', name="receiver_odk_post"),
-    url(r'^/(?P<app_id>[\w-]+)/$', 'post', name='receiver_post_with_app_id'),
+    url(r'^submission/?$',  'post', name="receiver_odk_post"),
```
Comment (dimagimon) [inline corehq/apps/receiverwrapper/urls.py]: E241 multiple spaces after ','

Comment (dimagimon) [inline corehq/apps/receiverwrapper/urls.py]: E241 multiple spaces after ','

**Comment by calellowitz** [comment]
I kicked the failed builds but one might have been real,
```
======================================================================

ERROR: test (corehq.apps.receiverwrapper.tests.test_app_id.TestAppId)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 232, in inner

    return helper(*args, **kwargs)

  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 205, in __call__

    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)

  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 220, in call_with_settings

    fn(*args, **kwargs)

  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 201, in fn_with_pre_and_post

    self.fn(*args, **kwargs)

  File "/mnt/commcare-hq-ro/corehq/apps/receiverwrapper/tests/test_app_id.py", line 37, in test

    self._test(self.build_id, self.app_id, self.build_id)

  File "/mnt/commcare-hq-ro/corehq/apps/receiverwrapper/tests/test_app_id.py", line 60, in _test

    form = FormAccessors(self.domain).get_form(form_id)

  File "/mnt/commcare-hq-ro/corehq/form_processor/interfaces/dbaccessors.py", line 111, in get_form

    return self.db_accessor.get_form(form_id)

  File "/mnt/commcare-hq-ro/corehq/form_processor/backends/couch/dbaccessors.py", line 56, in get_form

    return XFormInstance.get(form_id)

  File "corehq/ex-submodules/couchforms/models.py", line 139, in get

    doc = db.get(docid, rev=rev, **extras)

  File "/vendor/lib/python2.7/site-packages/couchdbkit/client.py", line 385, in open_doc

    docid = resource.escape_docid(docid)

  File "/vendor/lib/python2.7/site-packages/couchdbkit/resource.py", line 180, in escape_docid

    if docid.startswith('/'):

AttributeError: 'NoneType' object has no attribute 'startswith'
```

**Pull request closed without merging**